src/framework/MOM_io.F90

```diff
@@ -344,6 +341,14 @@ subroutine reopen_file(unit, filename, vars, novars, G, fields, threading, timeu
 !      !    filename,vars%name,name);
 !      !call MOM_error(NOTE,"MOM_io: "//mesg)
 !    enddo
+    !
```

Comment:
If I "call close_file(unit)" here, the model seg. faults later at line 895 of MOM_sum_output.F90 as if the array "CS%fields" was dissociated! 
It looks like fields(1:nvar) here is a pointer that goes away if the unit is closed!